template/packages/frontend/components/Title.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react"
+import styles from "./title.module.css"
+
+type Props = {
+  text: string
+}
+
+const Title = ({ text }: Props) => {
+  return (
+    <div>
+      <h2 className={styles.title}>{text}</h2>
+    </div>
+  )
+}
```

Comment:
If you use the special `children` prop then the component can be called like, `<Title>My Title</Title>`. The `React.ReactNode` type allows a string, a React element, an array, or a few other types that are allowed to be interpolated in JSX.

```suggestion
type Props = {
  children: React.ReactNode
}

const Title = ({ children }: Props) => {
  return (
    <div>
      <h2 className={styles.title}>{children}</h2>
    </div>
  )
}
```